pkgs/applications/display-managers/lightdm-slick-greeter/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, pkg-config
+, vala
+, intltool
+, autoreconfHook
+, wrapGAppsHook
+, lightdm
+, gtk3
+, pixman
+, libcanberra
+, libX11
+, libXext
+, linkFarm
+, lightdm_slick_greeter
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lightdm-slick-greeter";
+  version = "1.5.9";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = "slick-greeter";
+    rev = "f1527015df9d0315c0a2344de0a91bea1559484e";
```

Comment:
This shouldn't refer to a random git revision, but instead a release tag. If that does not exist, we follow the following format for `version`: `0.pre+<date>` where `<date>` is the date of the commit.